metis-common/src/main/java/eu/europeana/metis/mongo/MongoClientProvider.java

```diff
@@ -52,11 +52,17 @@
    * the details). The connection URL can provide settings that will override the default settings.
    *
    * @param connectionUri The connection URI as a string
+   * @param maxConnectionIdleTime The max connection idle time
```

Comment:
Hmm that is right! I think I've added seconds because I saw somewhere seconds being being specified but I guess I was wrong, so I've updated it to milliseconds as is the default.
It was used similarly in corelib with the MongoClientSettings.Builder so it's replicated here.

But come to think of it, I just remembered that this is not supported on the ConnectionString class with the url query parameter `maxIdleTimeMS` so I'm going to completely remove it from there and add a note when I pass it on to the api

And:
"A zero value indicates no limit to the idle time"